### ISSUE TITLE
return statement gets pulled to first column

### ISSUE THREAD

**Issue by TheRealAgentK** [jcberquist/commandbox-cfformat#83]
```
    package boolean function fghfgh(
        required numeric fgfg,
        required numeric fgfg
    ) {
        var fgfg = "";
        ```
        
    	<cfquery name="fgfg" datasource="#dsn#">
	    INSERT INTO fgfgfg 
	        (fgfg, fgfg, fgfg)
	    VALUES 
	    (<cfqueryparam value="#arguments.fgfg#" cfsqltype="cf_sql_integer">, <cfqueryparam value="#arguments.fgfg#" cfsqltype="cf_sql_integer">, 1)
	</cfquery>
        
        ```
        return true;
    }
```

On 0.15.6, cfformat complains and wants to move the return statement to the left into the first column (essentially remove all indent) of that line.

**Comment by TheRealAgentK** [comment]
Having an empty line between the closing tag island and the `return true;` makes the complaint go away.

**Comment by jcberquist** [comment]
This should be fixed in v0.15.8

**Comment by TheRealAgentK** [comment]
Yup, confirmed, thx!
